core/test/unit/models/base/base_spec.js

```diff
@@ -0,0 +1,28 @@
+var should = require('should'),
+    sinon = require('sinon'),
+    _ = require('lodash'),
+    models = require('../../../../server/models'),
+    ghostBookshelf,
+
+    sandbox = sinon.sandbox.create();
+
+describe('Models: base', function () {
+    before(function () {
+        models.init();
+        ghostBookshelf = _.cloneDeep(models.Base);
+    });
+
+    afterEach(function () {
+        sandbox.restore();
+    });
+
+    describe('setEmptyValuesToNull', function () {
+        it('should add function to prototype', function () {
+            ghostBookshelf.Model.prototype.setEmptyValuesToNull.should.be.a.Function();
+        });
+        it.skip('resets given empty value to null', function (done) {
```

Comment:
@kirrg001 This is giving me a headache... Not sure how I could test the functionality here... Can you help?

Reply:
What would you like to achieve and what isn't working?

Reply:
I want to unit test the fn itself. Do you think it's necessary?

Reply:
> and what isn't working?

Reply:
I wasn't able to stub the `get()` fn for the bookshelf model as it is needed [here](https://github.com/TryGhost/Ghost/pull/9432/files#diff-f8d02ad12b13d44b912bbf46cf7193b6R292)...

Reply:
You have to bind the model instance.
Right now you are binding the test context.